deployments/arbitrum/usdc.n/migrations/1686953660_configurate_and_ens.ts

```diff
@@ -0,0 +1,216 @@
+import { Contract } from 'ethers';
+import { expect } from 'chai';
+import { DeploymentManager } from '../../../../plugins/deployment_manager/DeploymentManager';
+import { diffState, getCometConfig } from '../../../../plugins/deployment_manager/DiffState';
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import { calldata, exp, getConfigurationStruct, proposal } from '../../../../src/deploy';
+import { applyL1ToL2Alias, estimateL2Transaction, estimateTokenBridge } from '../../../../scenario/utils/arbitrumUtils';
+
+const ENSName = 'compound-community-licenses.eth';
+const ENSResolverAddress = '0x4976fb03C32e5B8cfe2b6cCB31c09Ba78EBaBa41';
+const ENSSubdomainLabel = 'v3-additional-grants';
+const ENSSubdomain = `${ENSSubdomainLabel}.${ENSName}`;
+const ENSTextRecordKey = 'v3-official-markets';
+
+const arbitrumCOMPAddress = '0x354A6dA3fcde098F8389cad84b0182725c6C91dE';
+
+const cUSDTAddress = '0xf650c3d88d12db855b8bf7d11be6c55a4e07dcc9';
```

Comment:
Can remove both of these.

Reply:
Removed `cUSDT` one, but I think 
```
const setRewardConfigCalldata = utils.defaultAbiCoder.encode(
      ['address', 'address'],
      [comet.address, arbitrumCOMPAddress]
    );
```
still uses arbitrumCOMPAddress for reward config.